### PR TITLE
sql: increase index checkpoint interval to 2min

### PR DESCRIPTION
**Pull request by dt**
This ups the checkpoint interval from 10s to 2m.
This allows doing more work per checkpoint and rewrites the job less.
In the scale of a large backfill, two minutes of work is still a small amount to redo on node failure.

Release note: none.

### CODE REVIEW

File: pkg/sql/backfill.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ const (
 
 	// checkpointInterval is the interval after which a checkpoint of the
 	// schema change is posted.
-	checkpointInterval = 10 * time.Second
+	checkpointInterval = 2 * time.Minute
```
Comment (vivekmenezes) [inline pkg/sql/backfill.go]: I wish we could set this to schemachanger.lease.duration * schemachanger.lease.renew_fraction * 0.8 but I'm sure that will break some unittest.

**Review by eriktrinh — APPROVED** [review]
:lgtm:

> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/32657#-:-LSozkRCBp_84_NHbb0c:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained

File: pkg/sql/backfill.go (hunk 1 above)
Comment (dt) [reply pkg/sql/backfill.go]: Did you want me to try that? or just merge this one for now?

**Review by vivekmenezes — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/32657)* status: :shipit: complete! 1 of 0 LGTMs obtained

___
*[pkg/sql/backfill.go, line 61 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/32657#-LSMJEMRMQTVb5L9y2XG-r1-61:-LSplFtO1SlxkpSaWYxe:bbr661b) ([raw file](https://github.com/cockroachdb/cockroach/blob/ba80d7ba4a599fd7749434b6a589da6f1fd76993/pkg/sql/backfill.go#L61)):*
<details><summary><i>Previously, dt (David Taylor) wrote…</i></summary><blockquote>

Did you want me to try that? or just merge this one for now?
</blockquote></details>

I think this okay to merge as is.

**Pull request merged**